pkgs/games/among-sus/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, port ? "1234" }:
+
+stdenv.mkDerivation {
+  pname = "among-sus-unstable";
+  version = "2020-10-22";
+
+  src = fetchgit {
```

Comment:
I would really like to see a fetchSourceHut in nix...

Reply:
What benefits do you see that I don't?

Reply:
I think it would be nice to have. We don't necessarily need it.
It would save us from domain typos.

Reply:
Since that would result in a failing build I think we're pretty good. 

The only reason I know why we have fetchFromGitHub is that we can directly pull tarballs and don't have to fetch a git tree.

Reply:
Open a issue + PR for this, would be cool